src/state/pair/hooks.ts

```diff
@@ -1,38 +1,17 @@
 import { Pair } from '@kyberswap/ks-sdk-classic'
-import { Currency, Token } from '@kyberswap/ks-sdk-core'
-import { useCallback, useMemo } from 'react'
-import { useDispatch, useSelector } from 'react-redux'
+import { Currency } from '@kyberswap/ks-sdk-core'
+import { useMemo } from 'react'
+import { useSelector } from 'react-redux'
 
 import { PairState, usePair } from 'data/Reserves'
 
-import { AppDispatch, AppState } from '../index'
-import { Field, selectCurrency } from './actions'
+import { AppState } from '../index'
+import { Field } from './actions'
 
 export function usePairState(): AppState['pair'] {
   return useSelector<AppState, AppState['pair']>(state => state.pair)
 }
 
-export function usePairActionHandlers(): {
```

Comment:
unused. Besides, we already have `useSwapActionHandlers` function that contains `onCurrencySelection`